zwave/rootfs/etc/services.d/zwave/run

```diff
@@ -20,6 +20,33 @@ fi
 bashio::log.info "Starting OpenZWave..."
 bashio::net.wait_for 1883
 
+# Ensure local MQTT server has the right OZW status
+# Workaround for an incorrect retained OZW status in MQTT
+# https://github.com/home-assistant/hassio-addons/issues/1462
+OZW_STATUS=$(\
+    mosquitto_sub \
+        --host 127.0.0.1 \
+        --port 1883 \
+        --username ozw \
+        --pw "${MQTT_PASSWORD}" \
+        -C 1 \
+        -W 3 \
+        --retained-only \
+        --topic "OpenZWave/${OZW_INSTANCE}/status/" \
+)
+if bashio::var.has_value "${OZW_STATUS}" \
+    && [[ $(bashio::jq "${OZW_STATUS}" ".Status") != "Offline" ]];
+then
+    mosquitto_pub \
+        --host 127.0.0.1 \
+        --port 1883 \
+        --username ozw \
+        --pw "${MQTT_PASSWORD}" \
+        --retain \
+        --topic "OpenZWave/${OZW_INSTANCE}/status/" \
+        --message "$(bashio::var.json "Status" "Offline")"
+fi
```

Comment:
I think it's 3 time the same function, can we offload it as function `/usr/libs/mqtt_helper.sh` and include the function on the scripts to call it?

Reply:
I guess so.

Reply:
Done 👍